ocs_ci/ocs/ui/page_objects/object_details_tab.py

```diff
@@ -0,0 +1,89 @@
+from ocs_ci.ocs.ui.helpers_ui import logger
+from ocs_ci.ocs.ui.page_objects.storage_system_details import StorageSystemDetails
+from ocs_ci.ocs.ui.helpers_ui import extract_encryption_status
+
+
+class ObjectDetails(StorageSystemDetails):
+    def __init__(self):
+        StorageSystemDetails.__init__(self)
+
+    def get_encryption_summary(self):
```

Comment:
this function is almost identical to https://github.com/red-hat-storage/ocs-ci/pull/11043/files#diff-9f59a58d2d8e9dc2fb1605a4b1037132f54ce381ba0501feaf4af431a3a32f4eR264.
Please create common function/s for the similar part/s.
An option would be to create a function that will get either 'block+file' or 'object' and will perform the corresponding actions per provided value

Reply:
It’s a good idea to create a separate class for the encryption module in our POM.
Create a new class called EncryptionModule with the functions open, read, and close. Then, add this class to the inheritance of StorageSystemDetails using multiple inheritance (e.g., StorageSystemTab, <### EncryptionModule>).

This way, both BlockAndFile and ObjectDetails will automatically have the encryption module functions. If anything needs to change in the module, it will be easier for engineers to find and fix it in one place.

Reply:
As suggested I have created an EncryptionModule class and  moved the encryption summary check login there.